src/info-panel/InfoPanel.js

```diff
@@ -0,0 +1,85 @@
+import useBreakpoint from "@w11r/use-breakpoint";
+import PropTypes from "prop-types";
+import React from "react";
+import styled from "styled-components";
+import { ReactComponent as MenuOpenIcon } from "../assets/icons/menuOpen.svg";
```

Comment:
yeah apparently there is some library that's part of Create React App that lets you import these as inline SVG markup rather than just a path to src. This was necessary to allow me to style the icon with CSS in this component (to change the color)